data.sql

```diff
@@ -91,3 +91,235 @@ UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name IN ('Angemon', 'Boarmon');
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Agumon', '02-03-2020', 0, true, 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Gabumon', '11-15-2018', 2, true, 8);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
```

Comment:
* You are missing some [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following,Jun%208th%2C%202008.) for vets as it is required that you add these data below 👇 , kindly add them 🙏 

> Insert the following data for vets:

> Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
> Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
> Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
> Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

Reply:
:+1: Great work on implementing these changes :+1: